### ISSUE TITLE
Fails to access fo SMB/CIFS external storage with OC Login

### ISSUE THREAD

**Issue by Veter111** [owncloud/android#1805]
### Actual behaviour

When I try to get access to my files from external SMB/CIFS with OC Login, I see black page. 
But if I using iOS or desktop clients all working normally.
### Expected behaviour

I should like to see my files ))
### Steps to reproduce
1. Create external SMB/CISF storage with OC Login
2. Install Android client
3. Try to open mounted folder from client

Can this problem be reproduced with the official owncloud server?
(url: https://demo.owncloud.org, user: test, password: test)
### Environment data

Android version:
4.4.2
Device model: 
Samsung SM-T231 and some other
Stock or customized system:
Stock
ownCloud app version:
2.1.1
ownCloud server version:
9.1.0
#### Web server error log

```
[12/Sep/2016:18:28:23 +0300] "PROPFIND /owncloud/remote.php/webdav/%D0%A1%D1%83%D0%BD%D1%81%D1%83%D1%81%20%D1%82%D0%B5%D1%80%D1%80%D0%B8%D1%82%D0%BE%D1%80%D0%B8%D0%B8/ HTTP/1.1" 503 223
```
#### ownCloud log (data/owncloud.log)

```
{"reqId":"V9bJl3S4UNX3eVVQB20H1QAAAAc","remoteAddr":"XXX.XXX.XXX.XXX","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 503 Storage not available\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\ServiceUnavailable\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(903): OCA\\\\DAV\\\\Connector\\\\Sabre\\\\ObjectTree->getNodeForPath('\\\\xD0\\\\xA1\\\\xD1\\\\x83\\\\xD0\\\\xBD\\\\xD1\\\\x81\\\\xD1\\\\x83\\\\xD1\\\\x81 \\\\xD1\\\\x82...')\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/CorePlugin.php(334): Sabre\\\\DAV\\\\Server->getPropertiesForPath('\\\\xD0\\\\xA1\\\\xD1\\\\x83\\\\xD0\\\\xBD\\\\xD1\\\\x81\\\\xD1\\\\x83\\\\xD1\\\\x81 \\\\xD1\\\\x82...', Array, 0)\\n#2 [internal function]: Sabre\\\\DAV\\\\CorePlugin->httpPropFind(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#4 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(459): Sabre\\\\Event\\\\EventEmitter->emit('method:PROPFIND', Array)\\n#5 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(248): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#6 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(56): Sabre\\\\DAV\\\\Server->exec()\\n#7 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#8 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/lib\\\/Connector\\\/Sabre\\\/ObjectTree.php\",\"Line\":159,\"User\":\"dvv\"}","level":4,"time":"2016-09-12T15:28:23+00:00","method":"PROPFIND","url":"\/owncloud\/remote.php\/webdav\/%D0%A1%D1%83%D0%BD%D1%81%D1%83%D1%81%20%D1%82%D0%B5%D1%80%D1%80%D0%B8%D1%82%D0%BE%D1%80%D0%B8%D0%B8\/","user":"dvv"}
```

**Comment by michaelstingl** [comment]
@davivel A user reported this issue to me too. What do we need to further investigate?

00006332

**Comment by davivel** [comment]
@jvillafanez , is it possible we have a pre-built server where we can try to reproduce this?

**Comment by davivel** [comment]
Idea (not confirmed): source of the problem is that Android app authenticates every request, while iOS and desktop clients use a session cookie. Is it possible we already discussed this in other issue?

**Comment by michaelstingl** [comment]
What is the status here?

**Comment by davivel** [comment]
Sorry, not tested yet... need to find the time after holydays.

**Comment by michaelstingl** [comment]
> is it possible we have a pre-built server where we can try to reproduce this?

@davivel @jvillafanez Please collect all the missing infrastructure needed for testing and involve devops team. Perhaps also topic for @davitol and @SergioBertolinSG for the January meeting…

**Comment by jvillafanez** [comment]
We might need more logs.

In the config.php add the `wnd.logging.enable` key and reproduce the issue.
```
$CONFIG = array (
  'wnd.logging.enable' => true,
  'instanceid' => 'blablabla',
  'passwordsalt' => 'blablabla',
  .....
```
That should log more things about the SMB connection.

So far we only know that the storage isn't available, but no idea about the reason.

@Veter111 Could you check if it's possible to connect to the same external storage using the usual SMB connector instead of the SMB with OC login? If you could connect to the same external directory with the same account (use username + password authentication) there could be some issues with the user session.

**Comment by Veter111** [comment]
Hello team,
After upgrade to version 9.1.3, I can't reproduce this bug.
All working normally.

**Comment by michaelstingl** [comment]
@Veter111 Thanks for your feedback! 👍

**Comment by michaelstingl** [comment]
I've got user feedback, it has been fixed with 9.0.6 too.
